metrics.yaml

```diff
@@ -175,6 +175,27 @@ launch_event:
       source:
         description: Where the browser was launched from
         type: string
+  failed_browser_launch:
+    type: event
+    lifetime: ping
+    description: |
+      The browser did not launch due to a native messaging host fault. (Chromium Only)
+    bugs:
+      - https://linktobugs.page
+    data_reviews:
+      - https://linktodatareviews.page
+    notification_emails:
+      - gsherman@mozilla.com
+    expires: never
+    extra_keys:
+      command:
+        description: The command used within the NMH. LaunchFirefox/LaunchFirefoxPrivate
+      native_messaging_host:
+        description: The NMH that was used to launch the browser
```

Comment:
This should probably be a little more explanatory.

"The id of the NMH used to launch the browser, which equates to the build channel and only to the build channel."

With what it says there, my first read was that it could include the path to the browser executable, which is not data we should be tracking. Clarifying in the description might lead to fewer explanations about us tracking data we don't need to.